### ISSUE TITLE
Write in plain English,  Use Plain Language Guideline (positive sentences)

### ISSUE THREAD

**Issue by martynka** [w3c/coga#186]
Don't only talk about the plain language but also use it. 
https://www.plainlanguage.gov/
https://plainlanguage.gov/guidelines/concise/
https://www.sec.gov/pdf/handbook.pdf

Some examples include:

Write in the positive sentences:
Don't use negative compound: not able. Use single word: unable.
4.3.4.4 More Details, last sentence:
"The company will then not receive any feedback from groups who are not able to find the feedback button."
Write:
"The company will then not receive any feedback from groups who are unable to find the feedback button."

4.5.3.2 How it Helps, second last sentence:
"And even if they have completed this process in the past, they are not able to bring their experience into future purchases to anticipate the final price. "
write:
"And even if they have completed this process in the past, they are unable to bring their experience into future purchases to anticipate the final price. "

Write in the positive sentences:
Don't use negative compound: not … unless. Use single word:  only if 
4.4.4.2 Description, last sentence:
"Do not use metaphors and similes unless you include an explanation." 
Write instead:
"Use metaphors and similes only if you include an explanation."

4.5.1.2 Description
"Make sure controls and content remain in place and do not move, unless the user initiates the movement."
Write instead:
"Make sure controls and content remain in place and move, only if the user initiates the movement."

Write in the positive sentences:
Don't use negative compound:  not … until. Use single word: only when.
6.2.2 Amy Scenario 2, last sentence:
"When designing her own applications and websites she makes sure all the controls for animated objects or videos are very visible and do not start until the user has decided they wish to interact or view the object."
Write instead:
"When designing her own applications and websites she makes sure all the controls for animated objects or videos are very visible and start only when the user has decided they wish to interact or view the object."

**Comment by rachaelbradley** [comment]
Note: I've changed not able to unable but moving this to further editorial discussion.

**Comment by rachaelbradley** [comment]
Thank you for your great suggestions.  We made some but not all of these changes. We have also conducted several plain language edits and hope the results are easier to read. We appreciate the detailed comments.
